package.json

```diff
@@ -29,7 +28,7 @@
     "react-load-script": "^0.0.6",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
-    "react-static": "5.8.5",
+    "react-static": "^5.9.7",
```

Comment:
just curious, why the update?

Reply:
[because non-static routes were broken when running serve-build instead of serve-local](https://github.com/cityofaustin/janis/pull/263/commits/ad7cb36ec6fc1d2b324e0195554a9e1fb526e276)